fix/ownership.lua

```diff
@@ -0,0 +1,49 @@
+--@ module = true
```

Comment:
You should only mark the file as a module if it can actually be loaded as a module (that is, it does not have side effects when loaded). this line can be removed.